code/missionui/missionshipchoice.cpp

```diff
@@ -1929,6 +1930,11 @@ void commit_pressed()
 	if (!(The_mission.flags[Mission::Mission_Flags::No_briefing]))
 		gamesnd_play_iface(InterfaceSounds::COMMIT_PRESSED);
 
+	// plieblang - resume simulated speech if necessary
+	if (speech_init && !Player->auto_advance) {
```

Comment:
```suggestion
	if (speech_init() && !Player->auto_advance) {
```
`speech_init` probably needs to be a function call.

Reply:
I'm not sure how that compiled locally.
Anyway, it looks like `speech_init` creates a new voice instance whereas we only want to resume the current voice, so I've changed it in the new commit.